Tests/SwiftKuerySQLiteTests/TestParameters.swift

```diff
@@ -74,9 +74,9 @@ class TestParameters: XCTestCase {
                             XCTAssertEqual(rows![0][0]! as! String, "apple", "Wrong value in row 0 column 0")
                             XCTAssertEqual(rows![1][0]! as! String, "apricot", "Wrong value in row 1 column 0")
                             XCTAssertEqual(rows![2][0]! as! String, "banana", "Wrong value in row 2 column 0")
-                            XCTAssertEqual(rows![0][1]! as! Int32, 10, "Wrong value in row 0 column 1")
-                            XCTAssertEqual(rows![1][1]! as! Int32, 3, "Wrong value in row 1 column 1")
-                            XCTAssertEqual(rows![2][1]! as! Int32, -8, "Wrong value in row 2 column 1")
+                            XCTAssertEqual(rows![0][1]! as! sqliteInt, 10, "Wrong value in row 0 column 1")
```

Comment:
Could these use `as?` rather than `as!` (like you did in `testInt64()`) - if the returned type was different to `sqliteInt`, I think this would crash rather than failing.